docs/learn/tokenomics2/Inflation.md

```diff
@@ -2,33 +2,166 @@
-| Collators        | 3.06%      | 7.07        |
-| dApp reward      | 17.27%     | 39.93       |
-| Base Staker      | 23.04%     | 53.27       |
-| Ajustable Staker | 51.84%     | 119.85      |
\ No newline at end of file
+`BuildAndEarn` subperiod consists of one or more **eras**. Each era has a fixed length.
+`Voting` subperiod always consists of exactly **one era** but its length can be longer than a standard era length.
+
+### Example
+
+* Block is produced every 12 seconds
+* **Era length** is 7200 blocks which equals 24 hours (1 day)
+* `Voting` subperiod length is 10 eras
+* `Build&Earn` subperiod length is 81 eras
+* Cycle length is **4 periods**
+
+With such configuration, we'd end up with a cycle lasting 364 days (roughly a year), and each period taking around 3 months to complete.
+
+## Recalculation
+
+When a new cycle begins, soft-inflation cap is recalculated, and according to the calculated value, rewards for all network actors are adjusted.
+
+The reason why it's a _soft-cap_, instead of a _hard-cap_ is how staker & dApp rewards are distributed.
+Rewards are only minted when they are claimed, and it is possible that at the time of reward recalculation, some rewards remain unclaimed.
+As a result, theoretically, it's possible for the cycle inflation to exceed the _soft-cap_ even though in practice it will be highly unlikely
+due to various _burn_ mechanisms.
+
+## Inflation Distribution
+
+Based on the calculated _soft-cap_, rewards for all network participants are adjusted.
+
+### Collators
+
+_Collators_ get a fixed amount of the cycle's _soft-capped_ inflation.
+This amount is equally divided by the number of blocks in the cycle.
+
+$collator\_reward\_per\_block = \frac{total\_collator\_cycle\_reward}{blocks\_per\_cycle}$
+
+### Treasury
+
+Similar to the _collators_, treasury gets a fixed amount of the cycle's _soft-capped_ inflation.
+To avoid it being minted all at once, the amount is equaly divided by the number of blocks in the cycle.
+
+$treasury\_reward\_per\_block = \frac{total\_treasury\_cycle\_reward}{blocks\_per\_cycle}$
+
+### dApps
+
+dApp reward are _assigned_ at the end of each era during `Build&Earn` subperiod.
+This means that the total cycle's dApp reward amount has to be equally divided by the total number of `Build&Earn` eras in a cycle.
+
+$dapp\_reward\_pool\_per\_era = \frac{total\_dapp\_cycle\_reward}{number\_of\_cycles * eras\_per\_build\_and\_earn}$
+
+The dApp staking protocol will calculate how much each staked dApp should get.
+
+### Stakers
+
+There are two components to the staker rewards - regular _staking_ rewards & the _bonus_ reward for loyal stakers.
+
+#### Regular Staker Rewards
+
+Regular staker rewards are awarded for staking native currency, **ASTR**, on a dApp.
+These rewards have two components - the _base_ reward and the _adjustable_ reward.
+
+Base reward is the amount assigned to the reward pool regardless of how much has been staked in total.
+
```

Comment:
To be consistent with all other reward descriptions, add how is base reward calculated.

Base reward is the amount assigned to the reward pool _at the end of each era_ regardless of how much has been staked in total.

Reply:
True, confusing. I wrote _how_ but I really meant _when_ is reward caculated (end of each era)